rasa/nlu/schemas/nlu.yml

```diff
@@ -46,6 +46,10 @@ mapping:
         lookup:
           type: "str"
         examples: *examples_anchor
+  responses:
+    type: "map"
```

Comment:
I don't think that is possible since the keys are the intent names, e.g. `chitchat/ask_name` and they would need to be known to further validate.

I think in the future we should consider moving to strict yaml instead of normal yaml + validation

Reply:
I think it should work using https://pykwalify.readthedocs.io/en/unstable/validation-rules.html#matching-rule 🤔

Reply:
ah good point :+1: will add that in a separate PR as it needs to be added for the domain as well (I just copied it from there)